ISPP-Eventia/eventus
# Feature/#245 segmentar event list

close #245 

## changes

- improved the event list usage experience
- segmented the event list
- add visual improvements and fix behaviors generally

## steps to review

- open the app
- check the event list deeply
